source/js/components/petition/locale-strings.jsx

```diff
@@ -34,6 +34,7 @@ export default {
     "Yes, I want to receive email updates about Mozilla's campaigns.": `Yes, I want to receive email updates about Mozilla’s campaigns.`,
     "Add my name": `Add my name`,
     "Sign up": `Sign up`,
+    Subscribe: `Subscribe`,
```

Comment:
This string was introduced as the submit button in the newsletter sign up component added at the end of blog posts, but the actual string + translations were never registered in this file